infrastructure/controller/pkg/configmap-controller/create.go

```diff
@@ -0,0 +1,40 @@
+package configmapController
```

Comment:
I think I missed this when we introduced the package. That's not very golang in terms of conventions. 

I'd suggest something like `cmcontroller` or `cmctrl` (the guideline is short, abbreviated names https://blog.golang.org/package-names). I have no strong opinion about the name itself though